diff --git a/autoninja.py b/autoninja.py
index 539ee2ae..72ca730b
--- a/autoninja.py
+++ b/autoninja.py
@@ -157,6 +157,10 @@ def main(args):
   # starting and stopping reproxy.
   if not offline and use_remoteexec:
     ninja_path = os.path.join(SCRIPT_DIR, 'ninja_reclient.py')
+
+  if sys.platform.startswith('win'):
+    ninja_path = ninja_path.replace("\\", '\\\\')
+
   args = prefix_args + [sys.executable, ninja_path] + input_args[1:]
 
   num_cores = multiprocessing.cpu_count()
