diff --git a/autoninja.py b/autoninja.py
index 539ee2ae..72ca730b 100644
--- a/autoninja.py
+++ b/autoninja.py
@@ -180,6 +180,10 @@ def main(args):
   # Call ninja.py so that it can find ninja binary installed by DEPS or one in
   # PATH.
   ninja_path = os.path.join(SCRIPT_DIR, 'ninja.py')
+
+  if sys.platform.startswith('win'):
+    ninja_path = ninja_path.replace("\\", '\\\\')
+
   args = prefix_args + [sys.executable, ninja_path] + input_args[1:]
 
   num_cores = multiprocessing.cpu_count()
